core/build.gradle.kts

```diff
@@ -9,9 +9,64 @@ plugins {
   id("antlr")
   id("com.google.protobuf") version "0.8.17"
   id("com.diffplug.spotless") version "6.11.0"
+  signing
 }
 
-publishing { publications { create<MavenPublication>("maven") { from(components["java"]) } } }
+publishing {
+  publications {
+    create<MavenPublication>("maven-publish") {
+      from(components["java"])
+
+      pom {
+        name.set("Substrait Java")
+        description.set(
+          "Create a well-defined, cross-language specification for data compute operations"
+        )
+        url.set("https://github.com/substrait-io/substrait-java")
+        properties.set(mapOf("country" to "PE", "dsusanibar.type.of" to "Java"))
```

Comment:
What is this info used for?

Reply:
Sorry, that was for my internal test, let me delete that.